ZVV983/WeatherAnalyzer.Tests/WeatherAnalyzerProgramTests.cs

```diff
@@ -0,0 +1,116 @@
+using System.CommandLine.IO;
+using WeatherAnalyzer.Geocode;
+using WeatherAnalyzer.Geocode.Api;
+using WeatherAnalyzer.Geocode.Api.File;
+using WeatherAnalyzer.Util;
+using WeatherAnalyzer.Weather;
+using WeatherAnalyzer.Weather.Api;
+using WeatherAnalyzer.Weather.Api.File;
+
+namespace WeatherAnalyzer.Tests;
+
+public class WeatherAnalyzerProgramTests
+{
+    [Test, Order(1)]
+    public async Task DownloadWeatherForecastsAsync_InMemory_CreatesOutputFile()
```

Comment:
Ha order-re kell támaszkodnod, az nem a legszebb megoldás. Ez mehetett volna egy Setup-ba. Azt, hogy a fájl létezik-e azt nem biztos, hogy tesztelni kell. Pl tök valid, hogy a logika teszt elbukik a fájl nem léte miatt, de ez nézőpont kérdése.

Reply:
Az orderre azért volt szükség, mert a 2. teszt használta az 1. teszt által létrehozott fájlt, ami valóban nem a legszebb megoldás. A fájlt létrehozását áthelyezem a 2. teszt arrange részébe, ahova az való.